telethon/client/uploads.py

```diff
@@ -53,6 +53,7 @@ def _resize_photo_if_needed(
         # Don't use a `with` block for `image`, or `file` would be closed.
         # See https://github.com/LonamiWebs/Telethon/issues/1121 for more.
         image = PIL.Image.open(file)
+        exif = image.info['exif']
```

Comment:
Is this key guaranteed to exist or could it raise `KeyError`?

Reply:
Sorry, I fixed this in the second commit.